bin/docker-entrypoint

```diff
@@ -14,10 +14,14 @@ scheduler() {
   WORKERS_COUNT=${WORKERS_COUNT:-1}
   QUEUES=${QUEUES:-celery}
   SCHEDULE_DB=${SCHEDULE_DB:-celerybeat-schedule}
+  if [[ -n "$AUTO_RELOAD" ]]; then
+    WATCHER="watchmedo auto-restart --directory=./ --pattern=*.py --recursive --"
```

Comment:
Maybe only watch the code library (`redash/`) so it doesn't reload on tests?

Reply:
Maybe we should add a new entrypoint for dev-celery to avoid any confusion or issues that might result here?

Reply:
See efef063 and df2eb1d.